stripe/src/main/java/com/stripe/android/view/CardNumberEditText.java

```diff
@@ -185,13 +193,17 @@ public void afterTextChanged(Editable s) {
         });
     }
 
-    private void updateCardBrand(String partialNumber) {
-        @Card.CardBrand String oldBrand = mCardBrand;
-        mCardBrand = CardUtils.getPossibleCardType(partialNumber);
-        if (mCardBrand.equals(oldBrand)) {
+    private void updateCardBrand(@NonNull @Card.CardBrand String brand) {
```

Comment:
I'm guessing this is pre-existing code but it would be nice to use Enums instead of a String for `CardBrand`